pkg/tools/logging/schema.go

```diff
@@ -50,6 +50,14 @@ func getLogSchema(_ context.Context, _ mcp.CallToolRequest, req GetLogSchemaRequ
 			return nil, fmt.Errorf("could not find schema for log_type %s: %v", req.LogType, err)
 		}
 		return mcp.NewToolResultText(string(content)), nil
+	case "k8s_application_logs":
```

Comment:
Since this is mostly copy / paste of the above case, can you refactor to avoid the duplication?